OpenSim/Common/Component.cpp

```diff
@@ -615,86 +615,27 @@ void Component::setParent(const Component& parent)
 
 std::string Component::getFullPathName() const
```

Comment:
Seems reasonable to switch to `getAbsolutePathName()` (better now than later!) but make sure to change it everywhere (including comments) where we use/describe a Component's "full" path name.